### DISCUSSION TITLE
Discrepancy in Measured Capacity Loss and Degradation Contribution Values

### DISCUSSION THREAD

**Discussion by MirAbbasAli2A** [pybamm-team/PyBaMM#3502] [Q&A] [answered]
Hello PyBamm Team - 

I have found a discrepancy with the capacity loss evaluation. When I evaluate the measured capacity, the following curve shows ~0.1 Ah capacity loss from the beginning to the end of the test:

![image](https://github.com/pybamm-team/PyBaMM/assets/149210587/c1e6cba2-5366-4042-9558-33724390a9f3)

However, when I look at LLI contributions, I notice a capacity loss of up to 0.14 Ah:

![image](https://github.com/pybamm-team/PyBaMM/assets/149210587/1a325457-3b38-4186-a80a-b42f08b74b9f)

I know the difference is seemingly small (0.04 Ah) but can this just be explained by an accumulation of rounding error? Or is there something else in play?

FYI, I am using the DFN model with all aging options enabled:

model = pybamm.lithium_ion.DFN(options = {
    "thermal":"lumped",
    "dimensionality":0,
    "cell geometry": "arbitrary", 
    "SEI": "solvent-diffusion limited",
    "SEI porosity change": "true",
    "particle mechanics": ("swelling and cracking", "swelling and cracking"),
    "SEI on cracks": "true",
    "loss of active material": "stress-driven",
    "calculate discharge energy": "true",
    "lithium plating": "partially reversible",
    "lithium plating porosity change": "true"
    },
    name="TDFN",
    )

Thank you for your time.

**Comment by valentinsulzer** [comment, marked as answer]
Lithium inventory is not identical to capacity. Lithium inventory is the amount of cyclable lithium (Q_Li in [this notebook](https://docs.pybamm.org/en/latest/source/examples/notebooks/models/electrode-state-of-health.html)), capacity is the amount of charge you can draw from 4.2V to 2.5V at infinitely low C-rate (zero resistance). They are related but not equal. See the notebook I shared for details.

**Reply by MirAbbasAli2A** [reply]
Thank you for your prompt reply @tinosulzer . So just to confirm, the capacity loss definitions I provided below are converting the the loss in lithium inventory to a capacity loss and therefore are not identical to the capacity measured?

Q_SEI = sol["Loss of capacity to SEI [A.h]"].entries
Q_SEI_cr = sol["Loss of capacity to SEI on cracks [A.h]"].entries
Q_plating = sol["Loss of capacity to lithium plating [A.h]"].entries
Q_side = sol["Total capacity lost to side reactions [A.h]"].entries

**Reply by valentinsulzer** [reply]
Correct - although now I see the names are misleading! "capacity" here means "cyclable lithium capacity"

**Comment by yazyr** [comment]
@MirAbbasAli2A If you don't mind, can you provide me the code that you used to perform the capacity fade? I myself am trying to perform a simulation to see how capacity fades with respect to number of cycles and it would be a great deal of favour if you could share the program. Thanks, in advance.
